haystack/utils/deepsetcloud.py

```diff
@@ -865,9 +865,50 @@ def _get_evaluation_sets(self, workspace: Optional[str] = None) -> Generator:
         evaluation_set_url = f"{url}/evaluation_sets"
         return self.client.get_with_auto_paging(url=evaluation_set_url)
 
-    def _get_evaluation_set(
+    def upload_evaluation_set(self, file_path: Path, workspace: Optional[str] = None):
+        """
+        Uploads an evaluation set file.
+        The evaluation set will be accessible under the name of the uploaded file.
+
+        Currently only CSV files (having "," as delimiter) with the following columns are supported:
+        - question (or query): the question or query of the label
+        - text: the answer or relevant text to the question or query
```

Comment:
can we rename this param to "answer"? I know that if it's info retrieval then it's a passage of text, etc. but it's still an answer nevertheless. Unless it's not how you then name it in NLP?

Reply:
I agree with you, however that would need changes within DC and the annotation tool. I'd introduce that later on while allowing to fill in `text` too for compatibility reasons.
Btw I'm going to mention that you can upload files comming from the annotation tool if you chose CSV as export format.